db/repository/salesServiceItem.py

```diff
@@ -0,0 +1,49 @@
+from db.base import User, SalesServiceItem
+from sqlalchemy.orm import Session
+from fastapi import HTTPException,status
+
+def create(request, db: Session,current_user):
+    user = db.query(User).filter(User.username == current_user.username).first()
+    new_salesServiceItem = SalesServiceItem(name=request.name,price=request.price,uom_id=request.uom_id,category_id=request.category_id,created_user_id=user.id,updated_user_id=user.id)
```

Comment:
u can use dict spread operation like this.
new_salesServiceItem = SalesServiceItem(**request.dict())

Reply:
Will fix it.